### PR TITLE
Added FloraMC

### PR DESCRIPTION
**Pull request by ar2hh**
### General:
* [ ] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated metadata.json`)*
* [ ] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [ ] Folder with appropriate name has been created / updated (must not include any non-alpha characters or whitespace).
* [ ] No changes were made to the file's formatting (4 spaces per tab).
* [ ] There are no syntax errors.
* [ ] There is no pre-existing mapping matching my id (check if there is an existing folder).
* [ ] My field values match your requirements.
* You can view our patterns here: [metadata.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/metadata.schema.json), or take a look below and complete the field checklist:
  - [ ] `id`: a lowercase string, which should match the folder name *(ex. `myserver`)*
  - [ ] `name`: a string *(ex. `MyServerPvP`)*
  - [ ] `description`: hook between 16 and 80 characters *(ex. `Home of Competitive Minecraft PvP`)* 
  - [ ] `addresses`: an array with lowercase strings *(ex. `["my.server", "your-server.com"]`)*
    - You do not need to specify subdomains, Lunar Client services automatically detect them.
  - [ ] `primaryAddress`: the primary address that people connect to with (please include the subdomain if required) *(ex. `mc.my.server`)*
  - [ ] `minecraftVersions`: an array with Minecraft versions as strings *(ex. `["1.18", "1.19"]` - Must be versions supported by Lunar Client)*
  - [ ] `primaryMinecraftVersion`: a Minecraft version as a string *(ex. `1.18` - Must be versions supported by Lunar Client)*
  - [ ] `primaryColor`: a hexademical color code that primarily distiguishes the server *(ex. `#00FFFF`)* 
  - [ ] `secondaryColor`: a hexademical color code that accompanies the `primaryColor` of the server *(ex. `#FF0000`)*
  - [ ] `primaryRegion`: the primary region where your server operates in *(ex. `NA`)*
  - [ ] `regions`: a list of regions where you have servers located that service your players *(ex. `["NA", "EU", "AS"]`)*
  - [ ] `gameTypes`: a list of games that describe the content on your server, must be a max of 3 listed *(ez. `["PVP", "UHC", "HCF"]`)*

#### Logo
* [ ] My logo is a `png` file.
* [ ] I have uploaded my logo to my server folder *(ex. `lunarnetwork`)* and named it `logo.png`.
* [ ] My logo has a transparent background and is square (1:1 aspect ratio).
* [ ] My logo is `512` pixels in width and height.
* [ ] My logo is my own property and complies with relevant copyright/privacy laws.

#### Background
* [ ] My background is a `png` file.
* [ ] I have uploaded my background to my server folder *(ex. `lunarnetwork`)* and named it `background.png`.
* [ ] My background has an aspect ratio of 16:9.
* [ ] My background is atleast `1920` pixels in width and `1080` pixels in height (can be bigger, but must remain the same aspect ratio).
* [ ] My background contains relevant content pertaining to the server, is my own property, and complies with relevant copyright/privacy laws.
* [ ] My background doesn't contain any markings, text, or logos (unless part of a build).

### CODE REVIEW

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please confirm if a subdomain is required to connect to the server

If a subdomain is required to join the server, add this to the primaryAddress field ONLY.

**Comment by Arrshhh** [comment]
> Please confirm if a subdomain is required to connect to the server
> 
> If a subdomain is required to join the server, add this to the primaryAddress field ONLY.

No subdomain is required to connect to the server, The changes you have made are correct.

**Comment by Arrshhh** [comment]
No subdomain is not required to connect to the server, The changes you have made are correct.

**Review by itsJordan — CHANGES_REQUESTED** [review]
Could you please modify your description to be less than 80 characters long?

**Comment by Arrshhh** [comment]
Done

File: servers/floramc/metadata.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "primaryAddress": "floramc.club",
+    "minecraftVersions": [
+        "1.19",
+        "1.18",
+        "1.17",
+        "1.16",
+        "1.12",
+        "1.8",
+        "1.7"
+
```
Comment (itsJordan) [inline servers/floramc/metadata.json:17]:
```suggestion
```

**Review by itsJordan — APPROVED** [review]

**Pull request merged**